lib/index.js

```diff
@@ -5,6 +6,7 @@
 
 var integration = require('analytics.js-integration');
 var foldl = require('foldl');
+var analytics = require('analytics.js-core');
```

Comment:
`analytics` is available on the `integration` object, accessible within methods w/ 

```
this.analytics.user().id()
this.analytics.user().anonymousId();
//etc
```